data_base/Player.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "class": "Basic Player",
+    "maxHitPoint": 12,
+    "attackPoint": 1,
+    "inventory": {
+      "inventoryName": "Inventory5629",
+      "essence": {
+        "collectibleType": "Typetype",
+        "num": 12323
+      },
+      "artifact": {
+        "collectibleType" : "TypeTypeTypet"
+      }
+    },
+    "equipments": {
+      "weapon": {
+        "name": "sword",
+        "weaponAttackPoint": 1223
+      },
+      "armor": {
+        "name": "armor",
+        "dmgReduced": 122
+      }
+    },
+    "location": [0,0]
+  }
+]
```

Comment:
Is this an example of the player? Or the player start with the stats?

Reply:
this is just an example, you guys can tell me the starting stats or change it yourself.